StellarApp/Utilities/Protocols.swift

```diff
@@ -1,9 +1,3 @@
-//
-//  Protocols.swift
-//  StellarApp
-//
-//  Created by TingxinLi on 4/8/19.
-//  Copyright © 2019 TingxinLi. All rights reserved.
-//
-
-import Foundation
+protocol SideMenuDelegate {
```

Comment:
Needed?

Reply:
for this, we might need it because we were thinking of adding a side menu for a user to select the button. So before I remove it or did some change, I want to ask for matt first because he is the one to work on this part